templates/registration/login.html

```diff
@@ -4,26 +4,33 @@
 
     <h1>Connexion</h1>
     {% if user.is_authenticated %}
-        <p>Current user: {{ user.email }}</p>
+        <p>Vous êtes déjà connecté(e) avec l'e-mail : {{ user.email }}</p>
         <form action="{% url 'oidc_logout' %}" method="post">
             {% csrf_token %}
-            <input type="submit" value="logout">
+            <button class="fr-btn" type="submit" value="logout">Se déconnecter</button>
         </form>
-    {% else %}      
-        <form action="{% url 'oidc_authentication_init' %}" method="get">
+    {% else %}
+        <form action="{% url 'oidc_authentication_init' %}" method="get" id="init_oidc_form">
             <div class="fr-connect-group">
+                <p>Vous allez vous connecter à la suite numérique avec MonComptePro.</p>
+                <p>Si rien ne se passe, cliquez sur le bouton :</p>
```

Comment:
```suggestion
            <p>Vous allez vous connecter à la suite numérique avec MonComptePro.</p>
            <p>Si rien ne se passe, cliquez sur le bouton :</p>
            <div class="fr-connect-group">
```

je pense que c'est le style des paragraphes dans la class "fr-connect-group". Si tu sors les <p> les code à l'extérieur du div, ça devrait fonctionner, je pense. (j'arrive pas à tester sur ma machine parce que justement on reste pas ici)

Reply:
Ah si c'est bon, j'ai juste fait "retour" et c'est effectivement écrit normalement 👍🏽